vspace/src/main/webapp/WEB-INF/views/staff/spaces/spacelist.jsp

```diff
@@ -17,7 +17,8 @@
 			data-url="<c:url value="/staff/space/${space.id}?${_csrf.parameterName}=${_csrf.token}"/>"
 			data-call-on-success="<c:url value="/staff/space/list"/>"
 			data-call-on-error="<c:url value="/staff/space/list"/>"
-			data-toggle="modal" data-target="#confirm-delete"><span
+			data-warning="${targetSpacesList.contains(space.id) ? 'Warning! Other spaces have links to this space' : ''}"
```

Comment:
where is targetSpacesList coming from?

Reply:
It doesn't look like the warning part of the ticket is implemented?

Reply:
This should be an ajax call that checks if the space is linked from anywhere when the user actually tries to delete it. Otherwise me might run into performance issues for very big exhibitions, or someone could link a space while someone else is on the list page and there would be no warning.